emukit/bayesian_optimization/acquisitions/probability_of_improvement.py

```diff
@@ -67,3 +67,56 @@ def evaluate_with_gradients(self, x: np.ndarray) -> Tuple:
     @property
     def has_gradients(self):
         return isinstance(self.model, IDifferentiable)
+    
+    
+    
```

Comment:
I think the acquisition should also go in it's own file `probability_of_feasibility.py`

Reply:
agreed